### ISSUE TITLE
The new zone `America/Ciudad_Juarez` is missing

### ISSUE THREAD

**Issue by meowtec** [spencermountain/spacetime#395]
IANA:
```
    In the Mexican state of Chihuahua, the border strip near the US
    will change to agree with nearby US locations on 2022-11-30.
    The strip's western part, represented by Ciudad Juárez, switches
    from -06 all year to -07/-06 with US DST rules, like El Paso, TX.
    The eastern part, represented by Ojinaga, will observe US DST next
    year, like Presidio, TX.  (Thanks to Heitor David Pinto.)
    A new Zone America/Ciudad_Juarez splits from America/Ojinaga.
```

**Comment by spencermountain** [comment]
hey cool! Thank you.
will add this to the next release.

**Comment by facedoodle** [comment]
Hello @spencermountain , want to kindly ask when will next release be?

**Comment by spencermountain** [comment]
released as `7.4.6`
cheers
